ssv/spectest/tests/runner/duties/newduty/consensus_not_started.go

```diff
@@ -24,45 +24,43 @@ func ConsensusNotStarted() tests.SpecTest {
 		Name: "new duty consensus not started",
 		Tests: []*StartNewRunnerDutySpecTest{
 			{
-				Name:                    "sync committee aggregator",
-				Runner:                  startRunner(testingutils.SyncCommitteeContributionRunner(ks), &testingutils.TestingSyncCommitteeContributionDuty),
-				Duty:                    &testingutils.TestingSyncCommitteeContributionDuty,
-				PostDutyRunnerStateRoot: "29862cc6054edc8547efcb5ae753290971d664b9c39768503b4d66e1b52ecb06",
```

Comment:
remove post root completley?

Reply:
Now I want to work to bring them all back since it messes up tests.json

Reply:
it was added back